tools/toolchains/arm.py

```diff
@@ -440,15 +440,17 @@ def __init__(
             notify=None,
             macros=None,
             build_profile=None,
-            build_dir=None
+            build_dir=None,
+            coverage_patterns=None
     ):
         ARM.__init__(
             self,
             target,
             notify,
             macros,
             build_dir=build_dir,
-            build_profile=build_profile
+            build_profile=build_profile,
+            coverage_patterns=None
```

Comment:
Shouldn't need to change this - gcc-arm only

Reply:
please see my above comments, about why ARM and IAR need to be changed as well.